filestore/fs.py

```diff
@@ -187,7 +191,7 @@ def _db(self):
             for col_name in versioned_collection:
                 val = sentinel.find_one({'collection': col_name})
                 if val is None:
-                    raise RuntimeError('there is now version sentinel for '
+                    raise RuntimeError('there is now a version sentinel for '
```

Comment:
I think I led you astray with this suggestion. It looks like you actually mean, "There is *not a* version sentinel..."